backend/x11/output.c

```diff
@@ -137,8 +140,8 @@ struct wlr_output *wlr_x11_output_create(struct wlr_backend *backend) {
 	};
 	output->win = xcb_generate_id(x11->xcb_conn);
 	xcb_create_window(x11->xcb_conn, XCB_COPY_FROM_PARENT, output->win,
-		x11->screen->root, 0, 0, 1024, 768, 1, XCB_WINDOW_CLASS_INPUT_OUTPUT,
-		x11->screen->root_visual, mask, values);
+		x11->screen->root, wlr_output->width, wlr_output->height, 1024, 768, 1,
```

Comment:
The two values that were previously `0, 0` are x and y, I think those should remain 0. The two values that are `1024, 768` are the window size, which should be set to `wlr_output->{width,height}`.